lemur/destinations/service.py

```diff
@@ -28,6 +30,13 @@ def create(label, plugin_name, options, description=None):
             del option['value']['plugin_object']
 
     destination = Destination(label=label, options=options, plugin_name=plugin_name, description=description)
+    current_app.logger.info("Destination: %s created", label)
+
+    # add the destination as source, to avoid new destinations that are not in source, as long as an AWS destination
+    if plugin_name == 'aws-destination':
```

Comment:
Instead of hardcoding this, is there a way for use to allow a plugin to self-declare that it meets these requirements? Perhaps some metadata we can inspect?

Reply:
Today, we define `type` of either source/destination. Do we need a new type of `source+destination` we could key this logic off of?

https://github.com/Netflix/lemur/blob/master/lemur/plugins/bases/source.py#L13

Reply:
Good point Kevin, now a Destination-plugin can define if it qualifies as source. Also adopted this approach for the celery job that syncs destinations and sources based on this qualification. 
Thanks.